src/components/sbb-journey-summary/sbb-journey-summary.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { SbbJourneySummary } from './sbb-journey-summary';
+import { newSpecPage } from '@stencil/core/testing';
+import { InterfaceJourneySummaryAttributes } from './sbb-journey-summary.custom';
+
+const data: InterfaceJourneySummaryAttributes = {
+  config: {
+    legs: '',
+    vias: ['via'],
+    startPoint: '',
+    destination: '',
+    arrivalWalk: 0,
+    departure: { time: '2022-08-29T20:30:00' },
```

Comment:
can you check if we can use the current time here? Then we would atleast can test if the journey is today :)